### PR TITLE
Add/new haa encoder values

### PR DESCRIPTION
**Pull request by Bart-van-Ingen**
The HAA joints have been mechanically completed and have now been given the appropriate encoder values. They have been added to the controller and march4 .yaml files

### CODE REVIEW

File: march_hardware_builder/src/robots/march4.yaml [hunk 1]
```diff
@@ -21,21 +21,21 @@ march4:
...
+          slaveIndex: 6
+          encoder: {resolution: 12, minPositionIU: 3688, maxPositionIU: 4070, zeroPositionIU: 3858, safetyMarginRad: 0.05}
+
+    - left_hip_aa:
+        allowActuation: true
+        netNumber: 5
+        temperatureges: {slaveIndex: 8, byteOffset: 0}
+        imotioncube:
+          slaveIndex: 9
+          encoder: {resolution: 12, minPositionIU: 1010, maxPositionIU: 1401, zeroPositionIU: 1194, safetyMarginRad: 0.05}
```
Comment (Ishadijcks) [inline march_hardware_builder/src/robots/march4.yaml]: The resolution of the `right_hip_aa` is 382 IU, while the `left_hip_aa` is 391 IU, is this acceptable?

Comment (Ishadijcks) [reply march_hardware_builder/src/robots/march4.yaml]: In the same vein is the zero position of `right_hip_aa` 170 IU from its start, and `left_hip_aa` 184 IU

Comment (Ishadijcks) [reply march_hardware_builder/src/robots/march4.yaml]: Could just be that the `minPositionIU` of the `right_hip_aa` is 10 toO HIGH, or the `left_hip_aa` too low.

Comment (Bart-van-Ingen) [reply march_hardware_builder/src/robots/march4.yaml]: The min and max position of the joints was determined by pushing the joints to their physical min and max position. The difference in range is therefore caused by the hardware. This difference can therefore not be solved through a software solution.

**Comment by Ishadijcks** [comment]
Please fix the conflict

**Review by martijnvandermarel — APPROVED** [review]

**Pull request merged**